src/SetChunkData.cpp

```diff
@@ -6,6 +6,7 @@
 #include "Globals.h"
 #include "SetChunkData.h"
 #include "BlockEntities/BlockEntity.h"
+#include "Entities/Entity.h"
```

Comment:
cSetChunkData stores a vector of StoredEntity's so compilation fails without it.